tracking/translations_parser/cli/taskcluster.py

```diff
@@ -220,6 +231,29 @@ def boot() -> None:
     )
     parser.run()
 
+    if args.publish_group_logs:
```

Comment:
The approach looks good to me, but I think we have this code somewhere in the offline Taskcluster publisher. Should we reuse it?